Mobile-Engineer-Flutter.md

```diff
@@ -16,22 +23,17 @@ You will build and maintain our cross-platform app written using Flutter. We exp
 ## Qualifications
 
 - Kurio's values (Get sh*t done, Be Bold, Stay Curious, Excellence, Collaboration)
-- Able to write scalable mobile applications using Dart
-- Have knowledge in Flutter's life cycle
-- Have knowledge in Software Architecture such as MVP or MVVM. 
-- Have knowledge of Object-Oriented Programming (OOP)
-- Have at least one basic understanding of Android or iOS development
-- Have at least one basic understanding of Android Material Design or Cupertino Design principles
-- Have knowledge of SQL
-- Understand the very basic conecept of RESTful API
-- Possess strong debugging skills
+- Able to develop mobile applications using Dart
+- Knowledge in Flutter's life cycle
+- Familiar with REST API
+- Familiar with SQL
 - Familiar with Git workflow and its principles
-- Familiar with Agile and Scrum
 
 ## Preferred Qualifications
 
-- Understands reactive programming (RxDart) with BLoC pattern
+- Knowledge in Software Architecture such as MVP or MVVM
+- Good understanding of at least one of either Android or iOS development
+- Good understanding of reactive programming (i.e. RxDart with BLoC pattern)
 - Have performed either Unit or Integration Test
 - Deployed a Flutter application to the Play Store and App Store
-- Knowledge of CI/CD
```

Comment:
I think we should stick this one as "Preferred Qualifications" knowing that our recent issue with our automated release issue